src/site/content/es/authors/index.11tydata.js

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
hm, it feels not great needing to duplicate these .11tydata.js files for every language...